Jenkinsfile.release

```diff
@@ -0,0 +1,206 @@
+// -*- Groovy -*-
+/*
+  Require credentials:
+    - Microsoft Azure Service Principal
+    - Azure Storage Account
+
+  Maven docker image cannot be run with the root user
+  https://batmat.net/2017/01/30/do-not-run-your-tests-in-continuous-integration-with-the-root-user/
+  Docker container related to jenkins-jnlp require:
+  * uid 1000,
+  * gid 1000
+  * /home/jenkins/.jenkins
+
+  Jenkins Plugins:
+    * Azure-Credentials
+    * SSH-agent
+
+  docker image olblak/maven is just a customized docker image based on maven where maven is run as
+  user jenkins with uid 1000 and gid 1000, there is also a volume /home/jenkins/.jenkins defined in the image
+  otherwise the directory is created with root permission and Jenkins cannot add files there
+*/
+
+pipeline {
+    agent {
+      kubernetes {
+        label 'jenkins-release'
+        defaultContainer 'jnlp'
```

Comment:
```suggestion
```

Unnecessary I think.

Reply:
You were right and I delete from [here](https://github.com/jenkins-infra/release/blob/master/Jenkinsfile.d/release#L14-L11)